examples/HelloWorld/Program.cs

```diff
@@ -8,6 +8,7 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal;
```

Comment:
these internal ones may go away in 3.0

Reply:
Okay - that might cause complications with #11 - should try that next to know what to expect.